packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/container.ts

```diff
@@ -51,6 +60,11 @@ export class Container extends ServiceExtension {
    */
   public readonly trafficPort: number;
 
+  /**
+   * The default log group into which application container logs should be routed.
+   */
+  private _logGroup?: awslogs.ILogGroup;
```

Comment:
Is there any reason why we add underscore here? Or it is just to keep 2 names little different than each other?